### PR TITLE
Rename MLValue to OrtValue

### PR DESCRIPTION
**Pull request by snnn**

### CODE REVIEW

File: onnxruntime/core/optimizer/optimizer_execution_frame.h [hunk 1]
```diff
@@ -33,7 +33,7 @@ class OptimizerExecutionFrame final : public IExecutionFrame {
       return allocator_ptr_;
     }
 
-    const MLValueNameIdxMap& GetMLValueNameIdxMap() const noexcept { return ort_value_name_idx_map_; }
+    const OrtValueNameIdxMap& GetMLValueNameIdxMap() const noexcept { return ort_value_name_idx_map_; }
```
Comment (RyanUnderhill) [inline onnxruntime/core/optimizer/optimizer_execution_frame.h]: GetMLValueNameIdxMap()

File: onnxruntime/core/optimizer/optimizer_execution_frame.h [hunk 2]
```diff
@@ -33,7 +33,7 @@ class OptimizerExecutionFrame final : public IExecutionFrame {
...
     const std::unordered_map<int, const NodeArg*>& GetMLValueIdxNodeArgMap() const noexcept {
```
Comment (RyanUnderhill) [inline onnxruntime/core/optimizer/optimizer_execution_frame.h]: GetMLValueIdxNodeArgMap()

File: onnxruntime/test/framework/allocation_planner_test.cc [hunk 3]
```diff
@@ -219,7 +216,7 @@ class PlannerTest : public ::testing::Test {
     EXPECT_EQ(graph_.Resolve(), Status::OK());
     state_.SetGraphViewer(std::make_unique<GraphViewer>(graph_));
 
-    MLValueNameIdxMap& mlvalue_name_idx_map{state_.GetMLValueNameIdxMap()};
+    OrtValueNameIdxMap& mlvalue_name_idx_map{state_.GetOrtValueNameIdxMap()};
```
Comment (RyanUnderhill) [inline onnxruntime/test/framework/allocation_planner_test.cc]: mlvalue_name_idx_map

**Comment by snnn** [comment]
To minimize merging conflict, I'll fix the comments in the next PR.

**Pull request merged**